tutorials/desktop/windows-ubuntu-hyperv-containers/windows-ubuntu-hyperv-containers.md

```diff
@@ -0,0 +1,174 @@
+---
+id: tutorial-windows-ubuntu-hyperv-containers
+summary: CHANGE ME
+categories: desktop
+tags: tutorial,installation,windows,ubuntu,terminal,docker,containers,hyper-v
+difficulty: 5
+status: 
+published: 
+author: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
```

Comment:
I've added the missing metadata:

```
id: tutorial-windows-ubuntu-hyperv-containers
summary: Run Ubuntu containers with Hyper-V isolation on Windows 10 and Windows Server```
categories: desktop
tags: tutorial,installation,windows,ubuntu,terminal,docker,containers,hyper-v
difficulty: 5
status: Published
published: 2017-08-13
author: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
```